adapters/vrtcal/vrtcal.go

```diff
@@ -101,10 +101,17 @@ func getReturnTypeForImp(impID string, imps []openrtb2.Imp) (openrtb_ext.BidType
 				return openrtb_ext.BidTypeBanner, nil
 			}
 
+
 			if imp.Video != nil {
 				return openrtb_ext.BidTypeVideo, nil
 			}
 
+
+			if imp.Native != nil {
+				return openrtb_ext.BidTypeNative, nil
+			}
```

Comment:
Warning - This is an anti-pattern. This code assumes that for multi-format bid request, mediatype will be set to type by default. I would recommend to set the mediaType based on bidder response. Your bidder should be able to set `openrtb2.Bid.MType` field and use that field here to get mediaTypeForImp. [Here is an example](https://github.com/prebid/prebid-server/blob/f275f3287f644293d102c07be4306a6b81cf2642/adapters/aidem/aidem.go#L89) on how this is done. Even if your bidder does not support multi-format request, I would highly encourage you to make this change.

Reply:
Thank you for the feedback...We've updated to support the 2.6 mtype parameter when present....We've also included code to still support the pre 2.6 non passed mtype (being it wasnt in the oRTB standard until 2.6).